pkg/config/vars.go

```diff
@@ -117,21 +117,6 @@ func setKwokctlConfigurationDefaults(config *v1alpha1.KwokctlConfiguration) *v1a
 	}
 	conf.KwokVersion = addPrefixV(envs.GetEnvWithPrefix("VERSION", conf.KwokVersion))
 
-	if conf.KubeVersion == "" {
```

Comment:
This value will be used frequently in the following and needs to be initialized at the first

Reply:
Updated